_posts/0000-01-04-dhanyaramesh.md

```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![homer-agh](https://cloud.githubusercontent.com/assets/16547949/25400895/02672eca-29c2-11e7-87a5-3dc3d3110c29.jpg)
 {: .center}
 
-Caption here
+Caption here - Dhanya Ramesh
```

Comment:
This image is supposed to focus on something that you really want, like a 🌯 or a 🌮 😄